Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in imgix-rails.gemspec
 gemspec
+
+gem "rspec_junit_formatter", "~> 0.4.1"
```

Comment:
Should this be a dev dependency in the gemspec file?

Reply:
Sure - added it over there